### ISSUE TITLE
Equipment - Combine not able to do proper infantry attack

### ISSUE THREAD

**Issue by Wrangler9033** [MegaMek/megamek#48]
I was testing using industrial mechs, using the Harvesting Ant Mech. This industrial Mech mounts 2 Combines.  

The Combine is suppose to do 1D6 damage against infantry.  When i used the Mech, it was only doing 2 damage per hit.  The Rule is found on Page 146 Total Warfare rules book. Shows what effects it suppose to do, MegaMek isn't doing it.  Couple this be coded in?

**Comment by arlith** [comment]
Looks like this issue has opened up a can of worms.  I went ahead and implemented the 1d6 damage, it wasn't too hard.  However, when I ran some tests, I kept always getting the same damage.  Turns out it's because the damage was being reduced to a tenth, max 1.  This comes from the table on TW pg 216 (all physical attacks against infantry are reduced by 1/10).

Now, it turns out that if you look on TW pg 136, it lists the dual-saw as doing 1d6 damage to infantry and that it should ignore the 1/10th damage reduction, however the combine doesn't mention this at all.  I suspect that the table on TW pg 146 may be off.

I've opened up a rules clarification here: http://bg.battletech.com/forums/index.php?topic=53920.0
